packages/components/src/select-control/index.tsx

```diff
@@ -89,15 +115,17 @@ function SelectControl(
 				label={ label }
 				size={ size }
 				suffix={
-					<DownArrowWrapper>
-						<Icon icon={ chevronDown } size={ 18 } />
-					</DownArrowWrapper>
+					props.suffix || (
+						<DownArrowWrapper>
+							<Icon icon={ chevronDown } size={ 18 } />
+						</DownArrowWrapper>
+					)
 				}
+				prefix={ props.prefix }
```

Comment:
Is passing `prefix` here (and omitting it from the props passed to `<Select />`) a bug fix? If yes, should it be included in the changelog?

Reply:
Ditto here, I don't think it qualifies as a bug fix as it's just to support better typing. There was no bug before.